docs/wiki/deployment/process-auditing.md

```diff
@@ -188,6 +188,32 @@ If you would like to debug the raw audit events as `osqueryd` sees them, use the
 
 > NOTICE: Linux systems running `journald` will collect logging data originating from the kernel audit subsystem (something that osquery enables) from several sources, including audit records. To avoid performance problems on busy boxes (specially when osquery event tables are enabled), it is recommended to mask audit logs from entering the journal with the following command `systemctl mask --now systemd-journald-audit.socket`.
 
+### Avoid throttling, losing events and interpreting Audit publisher throttling messages
+
+If osquery is CPU constrained and is processing a high enough stream of events, you may receive this warning message:  
+`The Audit publisher has throttled reading records from Netlink for <N> seconds. Some events may have been lost.`.  
+This message can only appear at most every minute and it indicates that the Audit publisher had to slow down reading records from the Netlink socket for the reported duration, since the previous throttling message.
+This happens when osquery is not processing records fast enough to prevent its internal buffers growing too much and consuming too much memory.
+
+Throttling may cause loss of events, since the Audit subsystem backlog buffer could fill up; if that happens the kernel will be forced to drop some of them.  
+You can check if this is happening looking at the `lost` field via `audictl -s`.
```

Comment:
Should that be auditctl?

Reply:
@zwass Thanks for catching that!
I've fixed it and also rebased on latest master.